### PR TITLE
feat: Update the design to accomodate priority, status

### PR DESCRIPTION
**Pull request by pranavrajs**
Screenshot 2023-04-25 at 11 04 29 PM

### CODE REVIEW

**Review by muhsin-k — APPROVED** [review]

File: app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue [hunk 1]
```diff
@@ -330,7 +339,7 @@ export default {
       router.push({ path: frontendURL(path) });
     },
     onCardHover() {
-      this.hovered = !this.hideThumbnail;
+      // this.hovered = !this.hideThumbnail;
```
Comment (scmmishra) [inline app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue:342]: Is this intentional?

File: app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue [hunk 2]
```diff
@@ -12,92 +12,101 @@
...
-            <fluent-icon icon="person" size="12" />
-            {{ assignee.name }}
+            <input
+              :value="selected"
+              :checked="selected"
+              class="checkbox"
+              type="checkbox"
+              @change="onSelectConversation($event.target.checked)"
+            />
+          </label>
```
Comment (scmmishra) [inline app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue:30]: If [this](https://github.com/chatwoot/chatwoot/pull/6988/files#r1183405586) change is intentional, we could remove this or hide this permanently

**Review by iamsivin — COMMENTED** [review]
@pranavrajs I tested it locally and LGTM🎉. Found a small UI issue. 

If multiple labels are added to a conversation, the user interface becomes overcrowded, and the inbox name section experiences some display issues, as shown in the screenshot.

image

**Comment by pranavrajs** [comment]
> @pranavrajs I tested it locally and LGTM🎉. Found a small UI issue.

> If multiple labels are added to a conversation, the user interface becomes overcrowded, and the inbox name section experiences some display issues, as shown in the screenshot.

Noticed this on my end as well. I will add a fix for the same.

File: app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue (hunk 2 above)
Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue:29]: @scmmishra Not intentional, I have reverted the changes.

File: app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue]: Not intentional, I did it for testing. I have reverted the changes now.

**Comment by pranavrajs** [comment]
Closing this for now, I will come back with a Figma prototype before the implementation.

**Pull request closed without merging**